spec.emu

```diff
@@ -882,6 +957,217 @@ contributors: Google, Ecma International
       </emu-table>
     </emu-clause>
 
+    <emu-clause id="sec-temporal-parsemonthcode" type="implementation-defined abstract operation">
+      <h1>
+        ParseMonthCode (
+          _monthCode_: a String,
+        ): an Record with fields [[Number]] (an integer) and [[IsLeap]] (a Boolean)
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>
+          It parses a valid month code string (such as one returned by ToMonthCode or CreateMonthCode) into its parts.
+        </dd>
+      </dl>
+      <p>It performs the following steps when called:</p>
+      <emu-alg>
+        1. If the length of _monthCode_ is 3, then
+          1. Let _isLeap_ be *false*.
+        1. Else,
+          1. Assert: The length of _monthCode_ is 4.
+          1. Let _isLeap_ be *true*.
+        1. Let _monthCodeDigits_ be the substring of _monthCode_ from 1 to 3.
+        1. Let _number_ be ℝ(StringToNumber(_monthCodeDigits_)).
+        1. Return the Record { [[Number]]: _number_, [[IsLeap]]: _isLeap_ }.
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-temporal-createmonthcode" type="implementation-defined abstract operation">
+      <h1>
+        CreateMonthCode (
+          _number_: an integer,
+          _isLeap_: a Boolean,
```

Comment:
```suggestion
          _isLeapMonth_: a Boolean,
```

Might be good to distinguish from leap years vs. leap months. I know it's obvious to us, but for readers who are not familar wtih lunisolar calendars it might be clearer to emphasize that this is a leap month not a leap year.